config/services.php

```diff
@@ -52,6 +52,8 @@
 
     'google' => [
         'maps_api_key' => env('GOOGLE_MAPS_API'),
+        'maps_type_basemap' => env('GOOGLE_MAPS_TYPE_BASEMAP'),
```

Comment:
Please use sane defaults here as well - not everyone will go back in and update the .env file to have a value, and we don't want it to be nothing at all if the value is null or missing